### PR TITLE
Rectified most unintelligibles

### PR DESCRIPTION
**Pull request by EricBrianAnil**

### CODE REVIEW

File: jsparty/js-party-11.md [hunk 1]
```diff
@@ -186,7 +186,7 @@ It's not an open standard, so that means you can't freely implement this, if tha
 
 So \[unintelligible 00:21:40.07\] doesn't implement this, because it's not free as in free stuff, and so...
```
Comment (pnevares) [inline jsparty/js-party-11.md]: Can you update this to "Gecko" while you're in the file?

File: jsparty/js-party-11.md [hunk 2]
```diff
@@ -346,7 +346,7 @@ Another one -- I've got a whole bunch of free ones, as well. I've got more free
 
 It's totally all over the place - it's from webcam stuff, to speech detection, to creating speech, to just doing basic DOM stuff, understanding how event listeners work, and clicks, and ES6 and what not... And people seem to really like it, because it's just a great way of -- I don't know, it's kind of a neat way to learn modern Javascript.
 
-**Alex Sexton:** \[unintelligible 00:40:57.18\]
+**Alex Sexton:** Get a lot of surface air
```
Comment (pnevares) [inline jsparty/js-party-11.md]:
```suggestion
**Alex Sexton:** Get a lot of surface area
```

File: jsparty/js-party-11.md (hunk 1 above)
Comment (EricBrianAnil) [reply jsparty/js-party-11.md]: Yea sure :)

File: jsparty/js-party-11.md (hunk 2 above)
Comment (EricBrianAnil) [reply jsparty/js-party-11.md]: Ahh, thanks for that :)

**Comment by jerodsanto** [comment]
💚💚💚

@all-contributors please add @EricBrianAnil for doc

**Pull request merged**